pkg/kn/commands/route/human_readable_flags.go

```diff
@@ -72,10 +72,14 @@ func printRoute(route *servingv1alpha1.Route, options hprinters.PrintOptions) ([
 func calculateTraffic(targets []servingv1alpha1.TrafficTarget) string {
 	var traffic string
 	for _, target := range targets {
+		var tag string
+		if target.Tag != "" {
+			tag = fmt.Sprintf("(%s)", target.Tag)
+		}
 		if len(traffic) > 0 {
```

Comment:
It's just a matter of taste, but I prefer to remove duplicate effort like this:

```go
		format := fmt.Sprintf("%s%s=%d", tag, target.RevisionName, target.Percent)
		if len(traffic) > 0 {
			traffic += ", " + format
		} else {
			traffic = format
		}
```

Reply:
@toVersus 
Refined it in next patch. Thanks!